### PR TITLE
drop 0.10 support

### PR DESCRIPTION
**Pull request by dcousens**
Since it's dropped out of LTS,  lets follow suite

### CODE REVIEW

**Comment by dcousens** [comment]
ping @jprichardson

File: package.json [hunk 1]
```diff
@@ -51,8 +54,6 @@
     "bigi": "^1.4.0",
     "bip66": "^1.1.0",
     "bs58check": "^1.0.5",
-    "buffer-compare": "^1.1.0",
-    "buffer-equals": "^1.0.3",
```
Comment (dcousens) [inline package.json]: :tada:

**Comment by dcousens** [comment]
I assume this is also a breaking change actually?

**Comment by dcousens** [comment]
Integration tests are failing: 

>  Error: No unspent given

I'll let someone else cross verify though :)

**Comment by rubensayshi** [comment]
utACK

**Pull request merged**